Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
Nice, I see that this has been included in autoconf-archive  [1].
Actually, I don't know if the NM is based on that or inspired that macro.
I guess we should prefer the one from autoconf-archive.

[1] https://www.gnu.org/software/autoconf-archive/ax_compiler_flags.html